### ISSUE TITLE
Removing default css can lead to "An unhandled error has occurred."

### ISSUE THREAD

**Issue by wildermedeiros** [dotnet/AspNetCore.Docs#31939]
### Description

I believe it's worth pointing out in the docs about removing the project's default CSS, which can cause "An unhandled error has occurred." In my case, I removed it by accident and it caused this problem, after reverse debugging the project, I realized that I had removed, as in the example, `<link href="BlazingPizza.styles.css" rel="stylesheet" />` , which resulted in the error message being displayed, but there was no error, it was probably just a flag through the css.

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/blazor/fundamentals/handle-errors?view=aspnetcore-8.0&preserve-view=true

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/fundamentals/handle-errors.md

### Document ID

66a0b1c2-c45c-98ca-9808-6f340a861c44

### Article author

@guardrex

**Comment by guardrex** [comment]
Thanks @wildermedeiros ... I'll take a look at this; but due to extremely HIGH doc issue/PR churn, it will take a while to reach it. Thanks for the suggestion.

**Comment by guardrex** [comment]
I finally made it around to this. I think we'll leave what we have in place for a few reasons ...

* This doesn't generate a bonafide error ... the styles just aren't available, so the app goes all HTML ugly :smile:.
* We cover the file well at https://learn.microsoft.com/en-us/aspnet/core/blazor/components/css-isolation?view=aspnetcore-8.0#css-isolation-bundling ... what it's doing and all.
* There are like 100,000 ways one can break an app by deleting something that the project template or framework relies on ... even Merlin jumping on the keyboard while I'm working!!! 😸😆 ... ***which he has actually done!*** 😠 ... but seriously ... we can't cover all of the things that might go wrong.

Closing on that basis, but thanks for raising it for consideration! 🚀
